### PR TITLE
GP2 70 create project milestones client

### PR DESCRIPTION
**Pull request by markcsims**
https://asaphub.atlassian.net/browse/GP2-70

### CODE REVIEW

File: packages/gp2-components/src/organisms/__tests__/ProjectMilestone.test.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    status: 'Active',
+    ...overrides,
+  });
+  it('renders the title', () => {
+    const title = 'a milestone title';
+    const milestone = getMilestone({ title });
+    render(<ProjectMilestone milestone={milestone} />);
+    expect(screen.getByRole('heading', { name: title })).toBeInTheDocument();
+  });
+  it.each(gp2.projectMilestoneStatus)('renders the status - %s', (status) => {
```
Comment (jhonny-m) [inline packages/gp2-components/src/organisms/__tests__/ProjectMilestone.test.tsx:18]: 👍

File: packages/gp2-components/src/molecules/StatusPill.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  | gp2.ProjectResponse['status']
+  | gp2.ProjectResponse['milestones'][0]['status'],
+  { color: string; borderColor: string }
+> = {
+  Active: {
+    color: colors.info900.rgba,
+    borderColor: colors.info900.rgba,
+  },
+  Completed: {
+    color: colors.secondary900.rgba,
```
[the remark this answers is not in the document]
Comment (markcsims) [reply packages/gp2-components/src/molecules/StatusPill.tsx:23]: changed to success900.

File: packages/gp2-components/src/organisms/ProjectMilestones.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+});
+
+const ProjectMilestones: React.FC<ProjectMilestonesProps> = ({
+  milestones,
+}) => {
+  const minimumMilestonesToDisplay = 3;
+  const [expanded, setExpanded] = useState(false);
+  const getMilestoneListStyles = () => {
+    if (milestones.length < minimumMilestonesToDisplay + 1) {
+      return {
```
[the remark this answers is not in the document]
Comment (markcsims) [reply packages/gp2-components/src/organisms/ProjectMilestones.tsx]: done

File: packages/gp2-components/src/organisms/ProjectMilestones.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        ...rowStyles,
+        [crossQuery]: {
+          borderBottom: `1px solid ${colors.neutral500.rgb}`,
+        },
+      };
+    }
+    if (expanded) return rowStyles;
+
+    return { ...rowStyles, ...hideStyles };
```
[the remark this answers is not in the document]
Comment (markcsims) [reply packages/gp2-components/src/organisms/ProjectMilestones.tsx:56]: done

File: packages/gp2-components/src/organisms/ProjectMilestones.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  };
+  return (
+    <>
+      <Headline3 noMargin>Project Milestones ({milestones.length})</Headline3>
+      <div css={[contentStyles]}>
+        <Paragraph hasMargin={false} accent="lead">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```
[the remark this answers is not in the document]
Comment (markcsims) [reply packages/gp2-components/src/organisms/ProjectMilestones.tsx:63]: The ticket says `Static project milestones copy to be added.` - I think that's something we will add later.

**Review by jhonny-m — APPROVED** [review]

**Pull request merged**